fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -444,10 +448,11 @@ public int writeData(final long id, final int block, @Nonnull final byte[] value
      * Write stored fields document to the DB.
      * @param segmentName the segment name writing to
      * @param docID the document ID to write
-     * @param value the bytes value of the stored fields
+     * @param rawBytes the bytes value of the stored fields
      */
-    public void writeStoredFields(@Nonnull String segmentName, int docID, @Nonnull final byte[] value) {
+    public void writeStoredFields(@Nonnull String segmentName, int docID, @Nonnull final byte[] rawBytes) {
         byte[] key = storedFieldsSubspace.pack(Tuple.from(segmentName, docID));
+        byte[] value = serializer.encode(rawBytes);
```

Comment:
If this is running at the same time as older versions, will older versions of the code be unable to read the data that this writes?

Reply:
Added.